### PR TITLE
Added 27 new extras-openmoji/climate-environment

### PR DESCRIPTION
**Pull request by b-g**
Hi @carlinmack, 

as usual :) Could you have a 👀?

- `data/extras-openmoji.csv` check and enhance the names / tags
- general emoji design feedback

Many thanks!!

### CODE REVIEW

File: src/extras-openmoji/climate-environment/E2C0.svg [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<svg id="emoji" viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg">
```
Comment (carlinmack) [inline src/extras-openmoji/climate-environment/E2C0.svg]: Too big, don't need the white background for the sky, only the white shape for the iceberg, the shape of the water should be considered - maybe it should be the shape of the circle of the grid? 
image

File: src/extras-openmoji/climate-environment/E2C1.svg [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<svg id="emoji" viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg">
```
Comment (carlinmack) [inline src/extras-openmoji/climate-environment/E2C1.svg]: Also too big, maybe do the same circle for the background

File: src/extras-openmoji/climate-environment/E2C2.svg [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<svg id="emoji" viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg">
```
Comment (carlinmack) [inline src/extras-openmoji/climate-environment/E2C2.svg]: Also too big, maybe do the same circle for the background

File: src/extras-openmoji/climate-environment/E2C3.svg [hunk 4]
```diff
@@ -0,0 +1,52 @@
+<svg id="emoji" viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg">
```
Comment (carlinmack) [inline src/extras-openmoji/climate-environment/E2C3.svg]: too big, also probably recognisable if you're familiar but I thought it was an oil rig or similar

File: src/extras-openmoji/climate-environment/E2C4.svg [hunk 5]
```diff
@@ -0,0 +1,31 @@
+<svg id="emoji" viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg">
```
Comment (carlinmack) [inline src/extras-openmoji/climate-environment/E2C4.svg]: also big, only recognisable if familiar but it's fine 👍

File: src/extras-openmoji/climate-environment/E2CE.svg [hunk 6]
```diff
@@ -0,0 +1,24 @@
+<svg id="emoji" viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg">
```
Comment (carlinmack) [inline src/extras-openmoji/climate-environment/E2CE.svg]: wouldn't have the inner stroke of the meat connecting with the fork

File: src/extras-openmoji/climate-environment/E2CC.svg [hunk 7]
```diff
@@ -0,0 +1,40 @@
+<svg id="emoji" viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg">
```
Comment (carlinmack) [inline src/extras-openmoji/climate-environment/E2CC.svg]: too big

File: src/extras-openmoji/climate-environment/E2CB.svg [hunk 8]
```diff
@@ -0,0 +1,23 @@
+<svg id="emoji" viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg">
```
Comment (carlinmack) [inline src/extras-openmoji/climate-environment/E2CB.svg]: too big, not centered

File: src/extras-openmoji/climate-environment/E2C7.svg [hunk 9]
```diff
@@ -0,0 +1,31 @@
+<svg id="emoji" viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg">
```
Comment (carlinmack) [inline src/extras-openmoji/climate-environment/E2C7.svg]: too big, change shape of water to be segment of circle

File: src/extras-openmoji/climate-environment/E2C6.svg [hunk 10]
```diff
@@ -0,0 +1,20 @@
+<svg id="emoji" viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg">
```
Comment (carlinmack) [inline src/extras-openmoji/climate-environment/E2C6.svg]: too big, change shape of water to be segment of circle

File: src/extras-openmoji/climate-environment/E2D0.svg [hunk 11]
```diff
@@ -0,0 +1,19 @@
+<svg id="emoji" viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg">
```
Comment (carlinmack) [inline src/extras-openmoji/climate-environment/E2D0.svg]: sea should be segment of circle

File: src/extras-openmoji/climate-environment/E2D1.svg [hunk 12]
```diff
@@ -0,0 +1,23 @@
+<svg id="emoji" viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg">
```
Comment (carlinmack) [inline src/extras-openmoji/climate-environment/E2D1.svg]: All the blades should be evenly spaced around the center of the turbine

File: src/extras-openmoji/climate-environment/E2DA.svg [hunk 13]
```diff
@@ -0,0 +1,31 @@
+<svg id="emoji" viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg">
```
Comment (carlinmack) [inline src/extras-openmoji/climate-environment/E2DA.svg]: too big

File: src/extras-openmoji/climate-environment/E2CF.svg [hunk 14]
```diff
@@ -0,0 +1,32 @@
+<svg id="emoji" viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg">
```
Comment (carlinmack) [inline src/extras-openmoji/climate-environment/E2CF.svg]: would be good to connect the car and the fumes but not important to change

File: src/extras-openmoji/climate-environment/E2C0.svg (hunk 1 above)
Comment (carlinmack) [reply src/extras-openmoji/climate-environment/E2C0.svg]: also famously shouldn't there only be a tiny "tip" of the iceberg that's much smaller than the rest? I.E. it should be much more submerged

**Comment by b-g** [comment]
Hi @carlinmack BTW many thanks for the review! Super helpful! Will try to go over your points some time this week ...

File: src/extras-openmoji/climate-environment/E2C0.svg (hunk 1 above)
Comment (b-g) [reply src/extras-openmoji/climate-environment/E2C0.svg]: Thanks! Good ideas! Will make a tiny tip and try to make it a bit smaller ... however note we do have really "big" emoijs e.g. https://openmoji.org/library/#search=evening&emoji=1F306

File: src/extras-openmoji/climate-environment/E2C1.svg (hunk 2 above)
Comment (b-g) [reply src/extras-openmoji/climate-environment/E2C1.svg]: Not better with circle IMO ... will leave as it is. Sorry!

2020-02-27 at 10 27

File: src/extras-openmoji/climate-environment/E2CE.svg (hunk 6 above)
Comment (b-g) [reply src/extras-openmoji/climate-environment/E2CE.svg]: 2020-02-27 at 12 01

File: src/extras-openmoji/climate-environment/E2D1.svg (hunk 12 above)
Comment (b-g) [reply src/extras-openmoji/climate-environment/E2D1.svg]: 2020-02-27 at 12 13

File: src/extras-openmoji/climate-environment/E2CF.svg (hunk 14 above)
Comment (b-g) [reply src/extras-openmoji/climate-environment/E2CF.svg]: 2020-02-27 at 12 18

**Pull request merged**